### DISCUSSION TITLE
如何只在每篇文章的底部或者右侧边栏底部显示一个小广告?

### DISCUSSION THREAD

**Discussion by WenLiCG** [CaiJimmy/hugo-theme-stack#790] [General]
我用的是 stack 主题,尝试在 footer,[甚至直接在.md] 文中添加广告代码,结果出现的都是春联样的广告,两边各一个,底部一个大大的~~
有办法只在每篇文章的底部或者右侧边栏底部显示一个小广告?
感谢

**Comment by jsjcjsjc** [comment]
难搞哈
